mockall/src/lib.rs

```diff
@@ -1472,7 +1472,11 @@ impl Times {
             if self.range.0.end == 1 {
                 Err("should not have been called".to_owned())
             } else {
-                Err(format!("called more than {} times", self.range.0.end - 1))
+                Err(format!(
+                    "called {} times which is more than expected {}",
```

Comment:
I think this would sound more natural:
```suggestion
                    "called {} times which is more than the expected {}",
```

Reply:
I agree! Fixed in d1fffe3.